README.md

```diff
@@ -16,13 +18,16 @@ gh extension install dlvhdr/gh-prs
 
 ## Configuring
 
-Configuration is done in the `sections.yml` file under the extension's directory.
-Each section is defined by a top level array item and has the following properies:
+Configuration is provided within a `sections.yml` file under the extension's directory. If the configuration file is missing, a prompt to create it will be displayed to when running `gh prs`.
```

Comment:
```suggestion
Configuration is provided within a `sections.yml` file under the extension's directory. If the configuration file is missing, a prompt to create it will be displayed when running `gh prs`.
```